### ISSUE TITLE
On CommandBar with split and disabled buttons, enabled and disabled downward arrow are in same green color which creates confusion.

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#16339]
On CommandBar with split and disabled buttons in high contrast black mode, enabled and disabled downward arrow are in same green color which creates confusion.

### Environment Information

OS: Win 10
Browser:  Edge

### Describe the issue:

On CommandBar with split and disabled buttons in high contrast black mode, enabled and disabled downward arrow are in same green color which creates confusion.

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar

![external_down_arrow](https://user-images.githubusercontent.com/62233982/103512282-73cbf100-4e8e-11eb-9550-a61d67f16e84.PNG)

#### Actual behavior:
On CommandBar with split and disabled buttons, enabled and disabled downward arrow are in same green color which creates confusion.
#### Expected behavior:
On CommandBar with split and disabled buttons, enabled and disabled downward arrow should have different color to avoid confusion. Color may be white[for enabled one]

**Comment by ecraig12345** [comment]
Root cause of this is most likely the same as #16156.

**Comment by ecraig12345** [comment]
@andrefcdias Would you be willing to help with this one too? (On a second look, I think it's actually a different root cause than #16156.) You'll need to track down which component is used to render this button, and probably add some missing high contrast styling. Here's a codepen with an isolated example of the problem: https://codepen.io/ecraig12345/pen/vYXMvYY

**Comment by andrefcdias** [comment]
I'll take a look at it
